Add advantages, description, and hardware related documentation (#41)
PhotonVision/photonvision-docs: source/docs/hardware/cameras.rst

Supported Cameras
=================

> We support any USB camera, however, Logitech cameras dont work well with PhotonVision. We reccomend the Microsoft Lifecam HD-3000 Camera which is avaliable from `Andymark <https://www.andymark.com/products/microsoft-lifecam-hd-3000-camera>`_. Additionally, ELP webcam modules are also reccomended which can be found on Amazon, Ebay, Alibaba, or anything of the sort.

Reviewer: ```suggestion
We support any USB camera, however, Logitech cameras don't work well with PhotonVision. We recommend the Microsoft LifeCam HD-3000 Camera which is available from `Andymark <https://www.andymark.com/products/microsoft-lifecam-hd-3000-camera>`_. Additionally, ELP webcam modules are also recommended which can be found on Amazon, Ebay, Alibaba, or many other online sources.
```
Fix typos, rewording
